File: Construction and Analysis of Algorithms/programs/bubblesort.py
array = [1,6,10,2,3,5,8,4]

for i in range(len(array)):
    for j in range(len(array)-1-i):
        if(array[j] > array[j+1]):
            aux = array[j]
            array[j] = array[j+1]
            array[j+1] = aux

print(array)

File: Programming/Code/bubble-sort.c
/* Use bubble sort algorithm to sort 
elements inside a vector*/

#include <stdio.h>
#define QTT 10
int main(){
	int vector[QTT], aux;
	for (int i=0; i<QTT; i++){
		scanf("%d", &vector[i]);
	}

	for (int j=QTT-1; j>0; j--){
		for (int i=0; i<j; i++){
			if(vector[i] > vector[i+1]){
				aux = vector[i];
				vector[i] = vector[i+1];
				vector[i+1] = aux;
			}
		}
	}

	for (int i=0; i<QTT; i++){
		printf("%d ", vector[i]);
	}


	return 0;
}